### PR TITLE
Add link to peer review guide to readme

### PR DESCRIPTION
**Pull request by asteflova**
I came across a peer review guide used internally at Red Hat, but available externally as well, and thought it could make a nice addition to README. It's been around for years, and as far as I can see, the things it recommends are aligned with Foreman docs conventions.

I imagine it could serve as a heads-up to any potential contributor from outside the regular writing team that they might want to expect wording and structure of their contributions to change before merging. In addition, I also simply consider this to be a good peer-review resource to keep at hand.

However, I don't feel particularly emotionally attached to this PR, so feel free to discard it if you disagree. I would be totally okay with that.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.7/Katello 4.9 (planned Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.4 or older.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,11 +7,13 @@ This git repository contains PoC of improving Foreman documentation.
 ## Foreman Guides
 
 This is a tree of documentation based on Red Hat Satellite 6 official books.
-Contributions are welcome.
 See README in the guides/ subdirectory for more information.
 
 * [guides](guides/README.md)
 
+Contributions are welcome.
+New contributions are subjected to technical review for accuracy and editorial review for consistency. For an overview of what to expect from editorial review, see [Peer review guide for technical documentation](https://redhat-documentation.github.io/peer-review/#checklist).
```
Comment (ekohl) [inline README.md:15]: I'd consider moving this up to the main section or even introducing a `Contributing` section.

**Review by ekohl — COMMENTED** [review]
Lines 3 and 5 in this file could use an update. For Katello it's the official documentation now. For Foreman it isn't yet, but we could link to https://github.com/theforeman/foreman-documentation/milestone/3

**Review by maximiliankolb — APPROVED** [review]
I like it! Maybe also consider adding the link to the GH PR template if you think it makes sense/should be accessible when creating a PR.

**Comment by asteflova** [comment]
> I like it! Maybe also consider adding the link to the GH PR template if you think it makes sense/should be accessible when creating a PR.

@maximiliankolb Do you mean the PR template with a checklist for different doc versions? I personally had no trouble at all figuring it out when I submitted my first PR, so I think we can go without adding it to the readme. I wouldn't want the readme to be too overwhelming.

But if you still think it would be a good idea to add it, feel free to disagree with me and explain :)

**Comment by asteflova** [comment]
> Lines 3 and 5 in this file could use an update. For Katello it's the official documentation now. For Foreman it isn't yet, but we could link to https://github.com/theforeman/foreman-documentation/milestone/3

I updated the first section of the README to include all this.

**Review by maximiliankolb — APPROVED** [review]
LGTM; can also be expanded/changed later.

**Pull request merged**